provisioning_templates/provision/kickstart_default.erb

```diff
@@ -137,7 +137,7 @@ exec < /dev/tty3 > /dev/tty3
 #changing to VT 3 so that we can see whats going on....
 /usr/bin/chvt 3
 (
-<% if subnet.respond_to?(:dhcp_boot_mode?) -%>
+<% if subnet.respond_to?(:dhcp_boot_mode?) || @host.subnet6.nil? && @host.subnet6.respond_to?(:dhcp_boot_mode?) -%>
```

Comment:
How does this work? You test if `subnet6` is nil, and then call `respond_to?` on it?

Should this be:
```ruby
if subnet.respond_to?(:dhcp_boot_mode?) || (@host.subnet6.present? && @host.subnet6.respond_to?(:dhcp_boot_mode?))
```

And since `NilClass` (hopefully) doesn't `respond_to?` `:dhcp_boot_mode?` (but the call exists), this can actually just be:

```ruby
if subnet.respond_to?(:dhcp_boot_mode?) || @host.subnet6.respond_to?(:dhcp_boot_mode?)
```